app/views/ProfileView/components/DeleteAccountActionSheetContent/index.tsx

```diff
@@ -68,9 +68,14 @@ export function DeleteAccountActionSheetContent(): React.ReactElement {
 	);
 }
 
-const AlertText = ({ text = '' }) => {
+const AlertText = ({ text = '', marginTop = 0, marginBottom = 24 }) => {
```

Comment:
A normal `style` prop would make it more predictable than having to send props and understand something might not behave as expected because of a default prop value.

Reply:
Alright, I moved it to its own file and made the necessary adjustments.